frontend/src/scenes/persons/MergePerson.tsx

```diff
@@ -24,62 +25,48 @@ export function MergePersonButton({
-                        const newPerson = await api.create('api/person/' + person.id + '/merge/', {
-                            ids: selectedPeople,
-                        })
-                        if (newPerson.id) {
-                            toast('People succesfully merged into one.')
-                            posthog.capture('merge person completed')
-                            setIsModalOpen(false)
-                            onPersonChange(newPerson)
-                        }
-                    }}
-                >
-                    Merge all properties and events of the selected users into <strong>{person.name}.</strong>
-                    <br />
-                    <br />
-                    Note: If there is a conflict, the properties of <strong>{person.name}</strong> will take precedence.
-                    <Select
-                        mode="multiple"
-                        allowClear
-                        showSearch
-                        style={{ width: '100%' }}
-                        placeholder="Please select people to merge"
-                        onChange={(value) => setSelectedPeople(value)}
-                        filterOption={false}
-                        onSearch={(value) => {
-                            setListFilters({ search: value })
-                            loadPersons()
-                        }}
-                    >
-                        {persons.results &&
-                            persons.results
-                                .filter((p) => p.uuid !== person.uuid)
-                                .map((person) => (
-                                    <Select.Option value={person.id} key={person.id}>
-                                        {person.name}
-                                    </Select.Option>
-                                ))}
-                    </Select>
-                </Modal>
-            )}
-            <Button
-                onClick={() => {
-                    posthog.capture('merge person modal opened')
-                    setIsModalOpen(true)
+        <Modal
+            visible
+            title={'Merge ' + person.name}
+            onCancel={closeModal}
+            onOk={async () => {
+                const newPerson = await api.create('api/person/' + person.id + '/merge/', {
+                    ids: selectedPeople,
+                })
+                if (newPerson.id) {
+                    toast('Persons succesfully merged.')
+                    posthog.capture('merge person completed', { merge_count: selectedPeople.length })
+                    closeModal()
+                    onPersonChange(newPerson)
+                }
+            }}
+        >
+            Merge all properties and events of the selected users into <strong>{person.name}.</strong>
+            <br />
+            <br />
+            Note: If there is a conflict, the properties of <strong>{person.name}</strong> will take precedence.
```

Comment:
Might be not part of this change, but should we indicate this is not reversible?

Reply:
Not part of the PR, but I completely agree and it's a very minor change, I've included it.